api/src/main/java/jakarta/persistence/query/StaticNativeQuery.java

```diff
@@ -22,24 +22,30 @@
 import static java.lang.annotation.RetentionPolicy.RUNTIME;
 
 /**
- * Specifies a query expressed in the native SQL dialect of the database,
- * usually a SQL {@code SELECT} statement, which is executed by calling
- * {@link jakarta.persistence.Query#getResultList getResultList()},
- * {@link jakarta.persistence.Query#getResultStream getResultStream()}, or
- * {@link jakarta.persistence.Query#getSingleResult getSingleResult()}.
+ * Specifies a query expressed in the native SQL dialect of the database.
+ * This annotation may be applied to:
+ * <ul>
+ * <li>any method of any class or interface belonging to the persistence
+ *     unit, or
+ * <li>a query method or default method of a Jakarta Data repository with
```

Comment:
Mentioning default method here will cause confusion to users, who will be confused about why they would want to write a default method implementation and also add annotation to the method that defines the behavior.  At best, this will seem redundant (if the two match), and at worst result in uncertain behavior (if they don't match).  Leaving this out of the Javadoc will not stop Data providers who know what they ware doing from being able to implement the behavior of the annotation by turning the method into a default method that still has the annotation.

```suggestion
 * <li>a query method of a Jakarta Data repository with
```